benchmarks/src/sort_tpch.rs

```diff
@@ -294,7 +297,7 @@ impl RunOpt {
 
         let mut stream = execute_stream(physical_plan.clone(), state.task_ctx())?;
         while let Some(batch) = stream.next().await {
-            row_count += batch.unwrap().num_rows();
+            row_count += batch?.num_rows();
```

Comment:
I replaced `unwrap()` with `?` so that sort_tpch does not terminate early. Thanks for catching that !